corehq/apps/userreports/tests/test_report_builder.py

```diff
@@ -142,13 +146,15 @@ def test_builder_for_cases(self):
         self.assertEqual('first_name', first_name_prop.get_id())
         self.assertEqual('first name', first_name_prop.get_text())
 
-    @flag_enabled('SHOW_OWNER_LOCATION_PROPERTY_IN_REPORT_BUILDER')
-    def test_owner_as_location(self):
+    @patch('corehq.apps.userreports.reports.builder.forms.domain_has_privilege', return_value=True)
```

Comment:
@privilege_enabled did not work here for me. Not sure why, but also not going to fix it in this PR.